src/vs/platform/native/common/native.ts

```diff
@@ -109,6 +109,7 @@ export interface ICommonNativeHostService {
 
 	// Clipboard
 	readClipboardText(type?: 'selection' | 'clipboard'): Promise<string>;
+	writeClipboardData(data: any, type?: 'selection' | 'clipboard'): Promise<void>;
```

Comment:
type options are now repeated three times?  Worth add an enum?
`
const enum SelectionType {
	Selection = 'selection',
	Clipboard = 'clipboard'
}
`

Then we can:
`writeClipboardData(data: any, type?: SelectionType): Promise<void>;`